assembl/static2/js/app/pages/multiColumnsAdmin.jsx

```diff
@@ -1,9 +1,97 @@
 import React from 'react';
+import { compose, graphql } from 'react-apollo';
+import { connect } from 'react-redux';
+import { get } from '../utils/routeMap';
+import ExportSection from '../components/administration/exportSection';
+import Navbar from '../components/administration/navbar';
+import DiscussionPreferenceLanguage from '../graphql/DiscussionPreferenceLanguage.graphql';
 
 class MultiColumnsAdmin extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      exportLocale: null,
+      refetching: false,
+      translate: false
+    };
+  }
+
+  componentDidMount() {
+    this.props.router.setRouteLeaveHook(this.props.route, this.routerWillLeave);
+  }
+
+  componentWillUnmount() {
+    this.props.router.setRouteLeaveHook(this.props.route, null);
+  }
+
+  handleExportLocaleChange = (locale) => {
+    this.setState({ exportLocale: locale });
+  };
+
+  handleTranslationChange = (shouldTranslate) => {
+    this.setState({ translate: shouldTranslate });
+  };
+
   render() {
-    return <p>Administration de la phase MultiColumnsAdmin</p>;
+    const { section, languages, debateId } = this.props;
+    const { translate } = this.state;
+    const exportLocale = this.state.exportLocale || (languages && languages[0].locale);
+    const translation = translate && exportLocale ? `?lang=${exportLocale}` : '?';
+    const exportLink = get('exportThreadMulticolumnData', { debateId: debateId, translation: translation });
+    const currentStep = parseInt(section, 10);
+    return (
+      <div className="multicolumn-admin">
+        {section === '1' && (
+          // When the other sections of this admin will be added, this one will be the last section and not '1'
+          <ExportSection
+            withLanguageOptions
+            handleExportLocaleChange={this.handleExportLocaleChange}
+            handleTranslationChange={this.handleTranslationChange}
+            exportLink={exportLink}
+            translate={translate}
+            exportLocale={exportLocale}
+            languages={languages}
+            annotation="multicolumnAnnotation"
+          />
+        )}
+        {!isNaN(currentStep) && (
+          <Navbar currentStep={currentStep} totalSteps={1} phaseIdentifier="multiColumn" />
```

Comment:
Why show the Navbar when the admin currently only has 1 step? If it's for the future, I would put another check, 

```
const totalSteps = 1;
!isNan(currentStep) && totalSteps > 1 && ...
```

In future, if `totalSteps` increases, the code will be rendered, otherwise the navbar is left out.

Reply:
Unless you absolutely want a navbar to be present even in the case of 1 step

Reply:
At first I hadn't put it but I thought it made it clear that there is only one step of admin to deal with at the moment (1/1) and not that others are available but not displaying for any sort of reason (bug).

Reply:
Good idea